Makefile

```diff
@@ -58,7 +58,7 @@ all:	$(TARGETS)
 
 clean:
 	cd libopencm3 && make --no-print-directory clean && cd ..
-	rm -f *.elf *.bin
+	rm -r build_*
```

Comment:
type `make clean` 2 times. 
On the second invocation you will get (or on the first on fresh pull)
```
rm: cannot remove ‘build_*’: No such file or directory
make: *** [clean] Error 1
```
To have `make clean` not issue the error, change `-r` -fr`